WD-Firmware/rtos/rtosal/api_inc/rtosal_util_api.h

```diff
@@ -41,11 +41,6 @@
 /**
 * types
 */
-/* application specific initialization function */
-typedef void (*rtosalApplicationInit_t)(void *pParam);
-
-/* param error notification function */
-typedef void (*rtosalParamErrorNotification_t)(const void *pParam, u32_t uiErrorCode);
 
 /**
 * local prototypes
```

Comment:
since rtosalContextSwitchIndicationClear(void); will be moved to rtosal_util.c this header file will be removed

Reply:
header file removed - https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/f1d341104d2182f0324edf87bb22f94fa83dabc7